inmap/configExample.toml

```diff
@@ -36,6 +36,9 @@ SROutputFile = "${GOPATH}/src/github.com/spatialmodel/inmap/inmap/testdata/testS
 # Path to desired output shapefile location. Can include environment variables.
 OutputFile = "${GOPATH}/src/github.com/spatialmodel/inmap/inmap/testdata/output_${InMAPRunType}.shp"
 
+# Path to desired logfile location. Can include environment variables.
```

Comment:
Full sentences. (I realize the comment for OutputFile is not in full sentences; perhaps change that too?)